plugins/node/instrumentation-amqplib/src/utils.ts

```diff
@@ -19,10 +19,17 @@ import {
   diag,
   HrTime,
   Span,
-  SpanAttributes,
-  SpanAttributeValue,
+  Attributes,
+  AttributeValue,
```

Comment:
I think we want the Attributes and AttributeValue change targeting Next branch, separate from this PR. Looks like that change is [part of this issue](https://github.com/open-telemetry/opentelemetry-js/issues/4175).

Reply:
Actually that looks to be specifically pointing out packages in core that have a lower API version, and there may not be a next branch on contrib. 🤔

Reply:
I think that since the change was introduced in https://github.com/open-telemetry/opentelemetry-js-api/pull/142, and it was published with `@opentelemetry/api` version [1.1.0](https://github.com/open-telemetry/opentelemetry-js/blob/main/api/CHANGELOG.md#110-2022-01-25). then it is safe to replace the types here, as this package already has a peer dependency on `"@opentelemetry/api": "^1.3.0"` which is guarentee to include this change